### PR TITLE
ci for rust contract

### PR DESCRIPTION
**Pull request by janedegtiareva**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
   - NODE_ENV=ci-staging
 cache: yarn
+
+before_install:
+  - rustup target add wasm32-unknown-unknown
+  - nvm install 10.16
+  - nvm use 10.16
+
+install: 
+  - npm install
```
Comment (vgrichina) [inline .travis.yml]: Should use `yarn` given we use `yarn` everywhere else (including `cache: yarn`). yarn can be installed in `before_install` using `npm install -g yarn`

Comment (janedegtiareva) [reply .travis.yml]: done

**Comment by nearmax** [comment]
@vgrichina It is preferable if you had let @janedegtiareva to merge it. She was waiting for the feedback from me. 

@janedegtiareva We need to add to near-bindgen CI the test that checks whether rust examples are not broken.

**Comment by vgrichina** [comment]
> @vgrichina It is preferable if you had let @janedegtiareva to merge it. She was waiting for the feedback from me.

It's not like we cannot incorporate your feedback in follow ups. 
If I get review request and it passes CI, I assume it's good to merge if there isn't any specific comment like "DO NOT MERGE before @nearmax approval" or smth like that. It's generally much faster to move that way, especially given that I'm iterating on related stuff (GitPod).

**Pull request merged**